src/sync_endpoint.rs

```diff
@@ -0,0 +1,35 @@
+//! Endpoint -> Famedly Zitadel sync tool.
```

Comment:
The name `Endpoint` is quite generic. Would it make sense to give it a more specific name? Obviously not so specific as to name the partner, but maybe something that communicates that it's an email list we're getting from an endpoint?